notebooks/Notebook2.md

```diff
@@ -2,33 +2,56 @@
+
+Sadly I do not think I can simplify the tedious work that I had to do much more. You will see that there were some changes to the files for the player types this week. The changes allowed for simplification of various parts of the player types such as getting rid of the name aspect or adding simpler setters for each type. 
+
 
 ## Description
 
-**TODO:** Fill in this part with information about your work this week:
-important design decisions, changes to previous decisions, open questions,
-exciting milestones, preliminary results, etc. Feel free to include images
-(e.g., a sketch of the design or a screenshot of a running program), links to
-code, and any other resources that you think will help clearly convey your
-design process.
+This week I worked on a few main things such as: building the bride from input to data structure through a parser, Implementing some sort of single line input either via a physical java interface or that being the command line, and looking over and simplifying the player type classes with individual setters per input type vs one big setter that sets all input types. 
+
+I began the week by building a physical java interface for the system that would allow the user to input the language into a text box. This allowed me to easily test inputs and see if it was working properly. This went fairly smoothly until I tried to add a part to the language, this was a simple “start new game log” phrase that was evaluated in the interface portion of the code and created a new game. This would then allow the user to input the regular dsl language. However I kept getting a null pointer error and I could not figure out why because there were no null pointers being passed through. Eventually I figured out that when adding this extra piece of code to create a new game I was building a null game. I ended taking that portion out and it worked great from there on. I may end up going back and retrying something similar later. ([code here](https://github.com/hmc-cs111-spring2023/BBeckerArtifact/blob/main/src/Interface/TextInputInterface.java))
+
+Next I started to simplify the player type classes with individual setters per input type vs one big setter that sets all input types. This was fairly straightforward, I got rid of alot of bulk in the abstract player class and allowed for each player type to be created with just the input of the players number instead of name and number and all the other components that each player has. Then I built setters for each player's unique inputs. This allowed me to write something similar to the following:
+New PassingPlayer(playerOrNum).setYards(numYards).setINCs(outcome).setTDs(outcome).setINTs(outcome).updatePlayer();
+This was useful because I was able to handle inputs that are the same across different methods. ([code here](https://github.com/hmc-cs111-spring2023/BBeckerArtifact/tree/main/src/Player))
```

Comment:
I reviewed some of the files in this folder to look over how your player position classes are set up and I better understand why much of the code was previously described as tedious and how a call like the one you provided above functions. One recommendation I would make would be to rename some of the 'set' methods as 'add' methods (i.e. setINTs => addINTs) as it seems these methods just add to the current count. I don't think this really affects the outcome of what users will be interacting with but it is just a minor detail that could clarify what these functions do.